README.md

```diff
@@ -2,7 +2,7 @@
 
 *Checks your GitHub Pages site for common DNS configuration issues*
 
-[![Build Status](https://travis-ci.org/github/pages-health-check.svg)](https://travis-ci.org/github/pages-health-check) [![Gem Version](https://badge.fury.io/rb/github-pages-health-check.svg)](http://badge.fury.io/rb/github-pages-health-check)
```

Comment:
Replace with an Actions badge?

Reply:
Good idea! I added one after renaming the workflow so it would look nicer. 😄